src/main.cpp

```diff
@@ -47,6 +48,40 @@ static bool ends_with(const std::string &s, const std::string &suffix)
 	return ssz >= sufsz && !s.compare(ssz - sufsz, sufsz, suffix);
 }
 
+
+static int check_reg_files(lmp_options &opt, std::string &sql, std::string &crt)
+{
+	if (access(sql.c_str(), F_OK) && access(crt.c_str(), F_OK))
+		return 0;
+
+	if (!opt.force) {
+		cerr << "ERROR: Device already registered in "
+		     << opt.sota_dir << endl;
+		cerr << "Re-run with --force 1 to remove existing registration "
```

Comment:
Maybe we should advise a user to delete a device if he/she would like to re-register the device with the same name/ID (something like run `fioctl device delete`).

Reply:
At that point we are not sure about the situation at the backend. And we would have to tell about delete-denied as well. Perhaps a better location for that hint is a HTTP CONFLICT is returned on a registration attempt. But I would rather do improvements in that area as a separate effort, to avoid dragging the PR longer.